frontend/src/components/user/Header.tsx

```diff
@@ -30,44 +36,193 @@ const Header: React.FC<Props> = ({ actions: { logout } }): JSX.Element => {
+                  onClick={() => setIsOpen(!isOpen)}
+                  className="p-1.5 rounded-full hover:bg-orange-50 focus:bg-orange-100"
+                >
+                  <IoClose className="w-6 h-6" />
+                </button>
+              </div>
+            </div>
+            <nav className="border-t border-gray-100 pt-3">
+              <ul className="flex flex-col space-x-0 space-y-3">
+                {headerLinks?.map(({ name, href }: IHeaderLink, i: number) => (
+                  <li key={i}>
+                    {router.pathname.includes('/categories/questions') ? (
+                      <span className={classNames(
+                        'font-medium text-sm',
+                        router.pathname.includes(href) ? 
+                        'text-orange-500 font-bold' : ''
+                      )}>{name}</span>
+                    ) : (
+                      <Link href={href}>
+                        <a className={classNames(
+                          'font-medium text-sm hover:text-orange-500',
+                          router.pathname.includes(href) ? 
+                          'text-orange-500 font-bold' : ''
+                        )}>{name}</a>
+                      </Link>
+                    )}
+                  </li>
+                ))}
+                {router.pathname.includes('/categories/questions') ? (
+                  <span className="text-left text-sm font-medium">
+                    Logout
+                  </span>
+                ) : (
+                  <>
+                    {subLink?.map(({ name, href }, i) => (
+                      <Link 
+                        key={i}
+                        href={href}
+                      >
+                        <a
+                          className="text-left text-sm font-medium hover:text-orange-500"
+                        >
+                          {name}
+                        </a>
+                      </Link>
+                    ))}
+                    <button
+                      type="button"
+                      className="text-left text-sm font-medium hover:text-orange-500"
+                      onClick={logout}
+                    >
+                      Logout
+                    </button>
+                  </>
+                )}
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </nav>
+    </Transition>
+  )
+}
+
+function MenuToggle({ user, logout }: { user: IUser, logout: () => void }) {
+  const subLink = [
+    {
+      name: 'Profile',
+      href: '/profile'
+    },
+    {
+      name: 'Settings',
+      href: '/settings'
+    },
+  ]
+
+  return (
+    <Menu as="div" className="ml-3 relative">
+      <div>
+        <Menu.Button type="button" className="flex items-center">
+          <span className="sr-only">Open user menu</span>
+          <Avatar 
+            url={`${user?.avatar_url === null ? 'https://i.stack.imgur.com/l60Hf.png' : user?.avatar_url}`}
```

Comment:
use constant variable for default avatar `url`.

So since defaul avatar `url` is being used in other pages, I suggest to store this into a constant variable in `utils` so that it can just be imported rather than placing the whole url again

Reply:
So basically I fix in in the Update-Avatar branch but since you've mention it in here I will fix all the images that I already fix in the other branch